### DISCUSSION TITLE
Não consigo criar uma nova fork

### DISCUSSION THREAD

**Discussion by KevinCordeiro** [digitalinnovationone/dio-lab-open-source#38156] [Perguntas e Respostas] [answered]
Aparece a seguinte mensagem
Required fields are marked with an asterisk (*).
No available destinations to fork this repository.

**Comment by elidianaandrade** [comment, marked as answer]
Olá @KevinCordeiro! Isso ocorreu, pois você já criou um fork e o GitHub não permite múltiplos forks do mesmo repositório para a mesma conta.

 A depender do que deseja fazer, algumas das opções são: atualizar o repositório do fork já existente na sua conta; criar uma nova branch; ou apagar o repositório e criar novamente.

**Obs.:** Caso não encontre o fork na sua conta, logo abaixo desse erro o próprio GitHub indica o link que redireciona ao repositório na sua conta.
